Tests/PointerStar.Server.Tests/PointerStar.Server.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Majorsoft.Blazor.Components.Modal" Version="1.5.0" />
```

Comment:
Do we need this in the tests project? I would only expect it to be needed in the Client project